config/prod/htan-dcc-msk.yaml

```diff
@@ -1,7 +1,7 @@
 # Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html) that will
 # work with the s3-synapse-sync lambda: https://github.com/Sage-Bionetworks/s3-synapse-sync
 template_path: "remote/s3-synapse-sync-bucket.j2"
-stack_name: "htan-dcc-bu"
+stack_name: "htan-dcc-msk"
```

Comment:
you cannot rename to delete.  deletion must be a separate PR

Reply:
thanks, updated.